service/tablet_allocator.cc

```diff
@@ -895,7 +895,7 @@ class load_balancer {
         co_return all_colocated;
     }
 
```

Comment:
> tablet A: 7f261e1a (rack 2), d2e44609 (rack 1)

What are those hex numbers? I'm missing the terminology.

Reply:
> tablet A: 7f261e1a (rack 2), d2e44609 (rack 1)
> tablet B: 3a1425ef (rack 1), 8f261e1b (rack 2)
> 

Is this a legal starting point?

Won't the load balancer keep the rack ordering? @tgrabiec

Reply:
We sort replica sets of a DC by host id order, so the MSB of the node id are relevant for showing that we can arrive at that layout. The tablet allocator is also greedy, I think we don't have any guarantee replicas will be allocated always in the same rack order.

Reply:
I will improve description.